[AudioRoutingGroup] publish beta doc to v1 (#19623)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/audioroutinggroup.md

---
title: "audioRoutingGroup resource type"

> description: "The audio routing group stores a private audio route between participants in a multiparty conversation. Source is the participant itself and the receivers are a subset of other participants in the multiparty conversation."

Reviewer: ```suggestion
description: "The audio routing group stores a private audio route between participants in a multiparty conversation. The source is the participant itself, and the receivers are a subset of other participants in the multiparty conversation."
```

author: "hanknguyen"
ms.localizationpriority: medium
ms.prod: "cloud-communications"
doc_type: resourcePageType
---

# audioRoutingGroup resource type

Namespace: microsoft.graph

> The audio routing group stores a private audio route between participants in a multiparty conversation. Source is the participant itself and the receivers are a subset of other participants in the multiparty conversation.

Reviewer: ```suggestion
The audio routing group stores a private audio route between participants in a multiparty conversation. The source is the participant itself, and the receivers are a subset of other participants in the multiparty conversation.
```

> **Note:** [ConfigureMixer](../api/participant-configuremixer.md) does not involve any routes, it is for the entire call for setting the volume levels for source-receiver combinations.

## Methods

| Method                                                  | Return Type                               | Description                                  |
|:--------------------------------------------------------|:------------------------------------------|:---------------------------------------------|
| [Get audioRoutingGroup](../api/audioroutinggroup-get.md)| [audioRoutingGroup](audioroutinggroup.md) | Read properties and relationships of audioRoutingGroup object.|
| [Update](../api/audioroutinggroup-update.md)            | [audioRoutingGroup](audioroutinggroup.md) | Update receivers list.                       |
| [Delete](../api/audioroutinggroup-delete.md)            | None                                      | Delete the audio routing group.              |

> 

Reviewer: The methods table on the audioroutinggroup.md shows more operations. Please check.

Author: There is no list operation, and this is intended. The Id for an audio routing group is constructed using the source participant id and arg type. User can create and query for it.

## Properties

| Property      | Type              | Description                                                          |
| :----------   | :---------------- | :--------------------------------------------------------------------|
| id            | string            | Read-only.                                                           |
| receivers     | collection(string) | List of receiving participant ids.                                   |
| routingMode   | string            | Routing group mode.  Possible values are: `oneToOne`, `multicast`.   |
| sources       | collection(string) | List of source participant ids.                                      |

> > **Note:** Routing mode determines the restrictions on the sources and receivers. Only the following routing groups are supported.

Reviewer: Could we take this note out of the note format and leave it as regular text? @Lauragra

> - `oneToOne` - sources and receivers have only one participant each.

> > - `multicast` - source has one participant but there are multiple receivers. Receivers list may be updated.

Reviewer: ```suggestion
> - `multicast` - sources have one participant, but multiple receivers exist. The receivers list may be updated.
```

> > **Note:** If you create many audio routing groups (e.g. a bot per participant), only the audio of the top 4 dominant speakers is forwarded. It means even with customized audio routing group, if the speaker is not loud enough in the main mixer, he/she cannot be heard by the bot even if there is a private audio group just for this speaker and the bot.

Reviewer: ```suggestion
> **Note:** If you create many audio routing groups (e.g., a bot per participant), only the audio of the top 4 dominant speakers is forwarded. For example, if the speaker is not loud enough in the main mixer of a customized audio routing group, the bot will not hear it, even if there is a private audio group just for this speaker and the bot.
```
